app/models/job_invocation.rb

```diff
@@ -87,7 +89,17 @@ def notification_recipients_ids
   end
 
   def build_notification
-    UINotifications::RemoteExecutionJobs::BaseJobFinish.new(self)
+    # self.remote_execution_feature
```

Comment:
Was this left here intentionally?

Reply:
nope, I'll remove when you'll add more comments :-)